### ISSUE TITLE
label data of meta-validation

### ISSUE THREAD

**Issue by LYF14020510036** [hsukyle/cactus-maml#13]
During meta-training，labels are not used, but labels of all validation samples are used during meta-validation. You don't use these label data to adjust parameters, but to save the pre-trained model for the meta-testing. Do I understand correctly?

**Comment by kylehkhsu** [comment]
You're correct that we don't use the meta-training split labels. We only use the meta-validation labels to probe overfitting to the meta-training task distribution. The meta-validation labels have nothing to do with serializing the model for meta-testing.
